### PR TITLE
Support django4

### PR DESCRIPTION
**Pull request by dqfort**
Created a version for Django 4, tested also work in Django 3

### CODE REVIEW

File: django_mongoengine/mongo_admin/views.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def get_ordering(self, request=None, queryset=None):
 
     def _lookup_param_1_3(self):
         lookup_params = self.params.copy() # a dictionary of the query string
-        for i in (ALL_VAR, ORDER_VAR, ORDER_TYPE_VAR, SEARCH_VAR,
```
Comment (last-partizan) [inline django_mongoengine/mongo_admin/views.py:127]: This change looking suspicious. This is query for django-1.3, and i think it's better to just remove it, as we don't support 1.3, minimum version is 2.2.

**Comment by last-partizan** [comment]
everything else is looking good!

File: django_mongoengine/mongo_admin/views.py (hunk 1 above)
Comment (dqfort) [reply django_mongoengine/mongo_admin/views.py:127]: right, removed the own method. And I found the method get_filters return 5 values dict in Django 4, so I also changed the code. https://github.com/django/django/blob/aff79be03a8c787ba0cc7e00dcec9eeb8ad01c87/django/contrib/admin/views/main.py#L452

**Review by last-partizan — CHANGES_REQUESTED** [review]

**Review by last-partizan — APPROVED** [review]

**Comment by last-partizan** [comment]
Thanks, merged it.

**Pull request merged**